### PR TITLE
update objectives

### PR DESCRIPTION
**Pull request by tkellen**

### CODE REVIEW

File: workshop/exercises/nginx/objectives.md [hunk 1]
```diff
@@ -0,0 +1,4 @@
+- What is nginx?
+- Where does nginx store per-site configuration files?
+- What is an "upstream" in an nginx configuration?
+- How do you send traffic for web requests to a local service?
```
Comment (MattSurabian) [inline workshop/exercises/nginx/objectives.md]: *Could* add something about why we bother serving from NGINX vs express directly or whatever else.

File: workshop/exercises/ssh-key-passphrases/objectives.md [hunk 2]
```diff
@@ -1,3 +1,3 @@
-- Why is a SSH passphrases desirable?
+- Why is a SSH passphrase desirable?
```
Comment (MattSurabian) [inline workshop/exercises/ssh-key-passphrases/objectives.md]: +key: Why is a SSH key passphrase desireable?

File: workshop/exercises/nginx/objectives.md (hunk 1 above)
Comment (tkellen) [inline workshop/exercises/nginx/objectives.md]: :+1:

File: workshop/exercises/tasks/objectives.md [hunk 3]
```diff
@@ -1,5 +1,5 @@
 - What is an Ansible module?
 - How do you find documentation about Ansible modules?
 - How do you use an Ansible module in a task?
+- By default, how does ordering affect task execution in Ansible?
```
Comment (MattSurabian) [inline workshop/exercises/tasks/objectives.md]: Are we planning to touch on handlers? If so maybe we want another bullet point about that specifically?

**Comment by MattSurabian** [comment]
Small comments, address an merge away :+1:

**Pull request merged**